zipline-loader/src/androidMain/kotlin/app/cash/zipline/loader/cacheDatabaseAndroid.kt

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (C) 2022 Block, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package app.cash.zipline.loader
+
+import com.squareup.sqldelight.android.AndroidSqliteDriver
+import com.squareup.sqldelight.db.SqlDriver
+
+actual class DriverFactory(private val context: android.content.Context) {
+  actual fun createDriver(): SqlDriver {
+    return AndroidSqliteDriver(
+      schema = Database.Schema,
+      context = context,
+      name = "zipline-loader.db",
```

Comment:
This should probably be an argument to the class rather than hard-coded. It means we can't do things like in-memory tests or change the database based on API endpoint.